MDANSE/Tests/UnitTests/test_trajectory.py

```diff
@@ -189,28 +190,17 @@ def test_read_com_trajectory(self):
             allCoordinates.append(
                 [[0.0, 0.0, 0.0], [8.0, 8.0, 8.0], [4.0, 4.0, 4.0], [2.0, 2.0, 2.0]]
             )
-            conf = RealConfiguration(
-                self._chemicalSystem, allCoordinates[-1], allUnitCells[-1]
+            conf = PeriodicRealConfiguration(
+                self._chemicalSystem, allCoordinates[-1], UnitCell(allUnitCells[-1])
             )
             self._chemicalSystem.configuration = conf
             tw.dump_configuration(i)
 
         tw.close()
 
-        t = Trajectory(self._filename)
+        t = Trajectory(tf)
         com_trajectory = t.read_com_trajectory(
-            [0, 1, 2, 3], np.array([1.0, 1.0, 1.0, 1.0]), 0, 1, 1
+            self._chemicalSystem.atoms, 0, 1, 1
         )
-        self.assertTrue(np.allclose(com_trajectory, [[1.0, 1.0, 1.0]], rtol=1.0e-6))
+        self.assertTrue(np.allclose(com_trajectory, [[3.5, 3.5, 3.5]], rtol=1.0e-6))
```

Comment:
I had a closer look at this part since the expected result has changed in this one, but it is clear that the new values are correct.